### PR TITLE
[CSM-201] getLocale is broken after testReset

### PR DESCRIPTION
**Pull request by ksaric**
Fixed `getLocale` after `testReset` - https://issues.serokell.io/issue/CSM-201

### CODE REVIEW

**Comment by ksaric** [comment]
This fix should go into `sl-0.4-rc` also, right @akegalj ?

**Comment by akegalj** [comment]
> This fix should go into sl-0.4-rc also, right @akegalj ?
@ksaric this should go into sl-04. Ideally you would have rebased from 0.4-rc and fix would be there. Everything from `0.4-rc` will eventually go to master. This is especially important in 0.4 because its a *fix*. Please change base branche

File: src/Pos/Wallet/Web/State/Storage.hs [hunk 1]
```diff
@@ -58,8 +58,8 @@ instance Default WalletStorage where
     def =
         WalletStorage
         {
-          _wsWalletMetas = mempty
-        , _wsProfile = mzero
+          _wsWalletMetas  = mempty
+        , _wsProfile      = Just def -- FIXME: Use CProfile, not Maybe CProfile
```
Comment (akegalj) [inline src/Pos/Wallet/Web/State/Storage.hs]: does this really fixes `testReset` problem. Have you checked @ksaric ?

Comment (ksaric) [inline src/Pos/Wallet/Web/State/Storage.hs]: I checked @akegalj. I'll delete this branch and create it from `0.4-rc` base so I don't pull changes from `master`.

**Pull request closed without merging**